### PR TITLE
"Code Review" Bu 13

### PR DESCRIPTION
**Pull request by AndrewKorous-zz**
Wanted to merge to get credit despite functionality not being there, should still work

### CODE REVIEW

File: controllers/user.js [hunk 1]
```diff
@@ -150,5 +151,36 @@ module.exports = {
             });
         });
       });
-    }
+    },
```
Comment (kjarrett316) [inline controllers/user.js]: Bro, this comma should be removed. It can't be there.

File: public/javascript/controllers/usersController.js [hunk 2]
```diff
@@ -158,29 +177,33 @@ angular.module('bandApp')
                 .success(function(data){
                     $scope.user = data;
                     if($scope.user.local && $scope.user.local.state) $scope.user.local.state = $scope.user.local.state.toString();
-                    callback(null);
+                    callback();
```
Comment (kjarrett316) [inline public/javascript/controllers/usersController.js]: should be commented out??????????

**Comment by kjarrett316** [comment]
Code reviewed.

**Comment by Dreamwalker420** [comment]
That was a lot of files to read through, what exactly is this supposed to do again?

**Comment by AndrewKorous-zz** [comment]
Basically when an admin approves a pending new user, the server would then send an email to the new users email address notifying them of this. However, I never actually got to test and finish writing the code to the actual NodeMailer part cause the controller just straight up refused to post. 

So the parts that affected the functionality of the rest of the app are commented out.

**Comment by kwalter26** [comment]
Why so many comments?

**Pull request merged**